content/applications/inventory_and_mrp/maintenance/maintenance_requests.rst

```diff
@@ -0,0 +1,98 @@
+====================
+Maintenance requests
+====================
+
+.. |MO| replace:: :abbr:`MO (Manufacturing Order)`
+
+In order to keep equipment and work centers functioning properly, it is often necessary to perform
+maintenance on them. This can include preventive maintenance, intended to prevent equipment from
+breaking down, or corrective maintenance, which is used to fix equipment that is broken or otherwise
+unusable.
+
+In Odoo *Maintenance*, users can create *maintenance requests* to schedule and track the progress of
+equipment and work center maintenance.
+
+Create maintenance request
+==========================
+
+To create a new maintenance request, navigate to :menuselection:`Maintenance --> Maintenance -->
+Maintenance Requests`, and click :guilabel:`New`.
+
+Begin filling out the form by entering a descriptive title in the :guilabel:`Request` field (e.g.,
+"Drill not working").
+
```

Comment:
may want to add a quick mention about the 'Created By' field -- just for a more complete overview of the form